src/app/webbrowser/SettingsPage.qml

```diff
@@ -86,6 +86,24 @@ FocusScope {
                     ListItemLayout {
                         title.text: i18n.tr("Homepage")
                         subtitle.text: homepageListItem.currentHomepage
+                        Icon {
```

Comment:
Hmm, there's no feedback when you tap the icon, is there? Like a darker background until you release

Reply:
Aha, that'd be the use of an AbstractButton here

Reply:
@cibersheep 
1) you mean instead of the opacity ?
I've seen that at several other locations in the code the opacity of 0.3 is used for inactive items, I could use this value instead.
2) if I disable it, the user can click through it and the homepage dialog opens, which should IMO only happen on the left side.
@UniversalSuperBox  I've tried with AbstractButton (and ChromeButton) but without luck.

Reply:
e.g. in https://github.com/ubports/morph-browser/blob/xenial/src/app/ChromeButton.qml 1.0 / 0.3 is used

Reply:
> @cibersheep
>     1. you mean instead of the opacity ?
>        I've seen that at several other locations in the code the opacity of 0.3 is used for inactive items, I could use this value instead.

Fair enough :)
 
>     2. if I disable it, the user can click through it and the homepage dialog opens, which should IMO only happen on the left side.

I see, the tap gets detected by the ListItem. Yep. That would be more confusing